build.gradle

```diff
@@ -191,3 +191,9 @@ class Version {
         "$major.$minor.$revision${stage ? '.' + stage : ''}${snapshot ? '-SNAPSHOT' : ''}"
     }
 }
+
+test {
```

Comment:
Would it be better to create a separate Gradle task like `testNonIntermittent` for this purpose and point CI to run that task instead?

Reply:
Possibly, though as these tests are not ignored, they're still run locally if you want them to, so not sure adding a Gradle task for it would give us much?
I'm also hoping this category is short lived.